index.html

```diff
@@ -7,17 +7,18 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Jake's Itinerary</title>
     <link rel="stylesheet" href="/styles/main.css">
+    <link rel="icon" type="image/png" href="images/favicon1.png">
 </head>
 
 <body>
     <header><img class="pageimage" src="/images/patagonia.jpg" alt="patagoniaImg"><img class="logo" src="./images/Geologo.jpg" alt="Geologo" onclick="openNav()">
       
         <div id="mySidenav" class="sidenav">
             <a href="javascript:void(0)" class="closebtn" onclick="closeNav()">&times;</a>
```

Comment:
I would make this part of code a function,   target mySidenav this way if you have to change something globally it will be easier modify.